### PR TITLE
update pythonPackages.pep257

### PR DESCRIPTION
**Pull request by lsix**
###### Things done

- [X] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] Linux
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -5126,24 +5126,7 @@ in {
...
-     buildInputs = with self; [ pytest ];
-     propagatedBuildInputs = with self ; [ pep257 ];
-
-     meta = {
-       homepage = https://github.com/anderslime/pytest-pep257;
-       description = "py.test plugin for PEP257";
-       license = licenses.mit;
-     };
-  };
+  pytestpep257 = callPackage ../development/python-modules/pytestpep257.nix { };
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: its called `pytest-pep257`, let's rename it.
https://pypi.org/project/pytest-pep257/

Comment (lsix) [inline pkgs/top-level/python-packages.nix]: Done.

**Comment by FRidh** [comment]
also the filename?

**Comment by lsix** [comment]
done.

**Pull request merged**